.template/addons/hotwire/Gemfile.rb

```diff
@@ -0,0 +1,8 @@
+# Hotwire Rails
+insert_into_file 'Gemfile', after: /gem 'bcrypt'.*\n/ do
+  <<~RUBY.indent(2)
```

Comment:
```suggestion
  <<~RUBY
```

Hi @przbadu  and Thanks for your contribution! :) 🙌 

I've tested it locally, the prompt is nice :) What about removing the indent as these lines are not in a group? 

For illustration, here is the resulting indentation I had at the moment:
![image](https://user-images.githubusercontent.com/77609814/172994370-f8f510c0-3a01-4532-b8b4-7ae139a53f98.png)


Another suggestion open for your ideas, should we add a small comment before these two lines (something like `# Hotwire`) — asking @rosle @bterone  as well to share opinions :) 

Thanks!

Reply:
@malparty 

Thank you, yes those changes make sense to me.